lae_automation/test/strategies.py

```diff
@@ -368,12 +368,27 @@ def deployment_configuration():
         secretsfile=strategies.just(open(devnull)),
     )
 
+deployment_configuration = deployment_configurations
+
 
 def old_secrets():
+    def same_introducer_furl(config):
+        config["storage"]["introducer_furl"] = config["introducer"]["introducer_furl"]
+        return config
```

Comment:
Hopefully the comment lower down in `old_secrets` explains why this is here now.